configure: Add site-url when registering pipeline

mamedin opened the pull request:
Connects to https://github.com/archivematica/Issues/issues/1100

Commit:
configure: Add site-url when registering pipeline

sromkey commented:
ping to reviewers! @amayita @scollazo @hakamine @DavidHume-ArtefactualSystems @luisilara :)

hakamine commented on tasks/configure.yml:
```diff
@@ -138,6 +138,7 @@
       --username="{{ archivematica_src_configure_am_user }}"
       --password="{{ archivematica_src_configure_am_password }}"
       --email="{{ archivematica_src_configure_am_email }}"
+      --site-url="{{ archivematica_src_configure_am_site_url | default ('127.0.0.1') }}"
```
since `archivematica_src_configure_am_site_url` is defined in `defaults/main.yml` the default('127.0.0.1') filter may not be needed?

hakamine reviewed (approved):
LGTM. I added a comment (the default('127.0.0.1') filter in the ansible task may not be needed as the variable is defined in `defaults/main.yml`, but I think it won't affect the outcome of the task wither having it or not )

mamedin commented:
> LGTM. I added a comment (the default('127.0.0.1') filter in the ansible task may not be needed as the variable is defined in `defaults/main.yml`, but I think it won't affect the outcome of the task wither having it or not )

I added the default `127.0.0.1` for barckward compatibility. Probably the already deployed servers without the new `archivematica_src_configure_am_site_url` don't need to register the pipeline again but the vars.yml could be used as template.

mamedin commented:
> > LGTM. I added a comment (the default('127.0.0.1') filter in the ansible task may not be needed as the variable is defined in `defaults/main.yml`, but I think it won't affect the outcome of the task wither having it or not )
> 
> I added the default `127.0.0.1` for barckward compatibility. Probably the already deployed servers without the new `archivematica_src_configure_am_site_url` don't need to register the pipeline again but the vars.yml could be used as template.

I need a coffee. Yep, it is already defined in `default/vars,yml` so I'm going to remove it. Thanks @hakamine !
